packages/react-sdk/src/utils/attributes.ts

```diff
@@ -0,0 +1,146 @@
+// Sourced from https://github.com/hatashiro/react-attr-converter/blob/master/index.js
+const map: Record<string, string> = {
```

Comment:
1.) It needs a descriptive name, not just the type. It's like calling it `string` or `array`.
2.) I think it should have the same attributes as what will actually be in the tree (https://github.com/pantheon-systems/pantheon-content-cloud/pull/242/files#diff-79c4b71b942ce77430ca6b70d753e9996bd79aae1840cf6369c82b8d11150514R21) or at most what's actually in the tree + a few that you foresee being there in the near future.